makefile

```diff
@@ -301,8 +303,6 @@ $(buildDir)/output.%.coverage:$(buildDir)/test.% .FORCE
 $(buildDir)/output.%.coverage.html:$(buildDir)/output.%.coverage
 	go tool cover -html=$< -o $@
 # end test and coverage artifacts
-build-alltests: $(testBin)
```

Comment:
I think @richardsamuels added these for a reason (check if all tests compile locally).

Reply:
it's still there, just moved up to line 188. 

having said that `make dist-test` does ~roughly the same thing.